app/tag-management/all-tags/all-tags.component.css

```diff
@@ -0,0 +1,11 @@
+.page-header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border-style: none;
+}
+
+/deep/ .md-tab-label {
```

Comment:
This part was written by me.
/deep/ (or >>>) is a special Angular CSS selector which allows to apply styles beyond Angular's CSS encapsulation. This was the only way to make it work.